### PR TITLE
custom calotowers for collapsed scenario and hlt

### PR DESCRIPTION
**Pull request by deguio**
this PR complements what was done by @abdoulline in #21810 and takes care of setting thresholds to old values for HLT

@kpedro88 @mariadalfonso @hatakeyamak

### CODE REVIEW

**Comment by kpedro88** [comment]
@deguio the intention in #21842 is not to construct CaloTowers from collapsed rechits, but rather the uncollapsed prerechits, so I'm not sure this makes sense (at least for offline...)

**Comment by kpedro88** [comment]
I guess so. I'm not sure things can possibly come out exactly right at HLT with this scheme. The inner/outer energy assignment and tower 28/29 splitting in CaloTowers both rely on HcalTopology, which uses the real segmentation rather than the fake collapsed segmentation. I don't think it's possible to support both at the same time (and removing knowledge of the real segmentation will surely break more things downstream).

**Comment by deguio** [comment]
ok. I guess we will discover how much that's a problem from the pre4 validation where we plan to have collapsed vs uncollapsed compared. let's see..
we will always be in time to go uncollapsed everywhere  ;)

**Comment by kpedro88** [comment]
please test

**Comment by kpedro88** [comment]
Do you actually need the import?

**Comment by abdoulline** [comment]
@deguio  @kpedro88 
Sorry if I'm missing something. 
In fillDescription of towermaker (CaloTowerCreator) new additional cuts ("...threshold1")= 0.8,  and in HLT menu old cuts ("...threshold") are also 0.8
So shouldn't it be enough for current HLT menus,  which only _contain old_ threshold(s) ?

**Comment by deguio** [comment]
hi salavat,
my understanding is that for the HLT step in relvals, where the 2018 era is used, you need this customization function to put the old values back given that the collapser will be running

File: HLTrigger/Configuration/python/customizeHLTforCMSSW.py [hunk 1]
```diff
@@ -17,6 +17,17 @@
 #                     pset.minGoodStripCharge = cms.PSet(refToPSet_ = cms.string('HLTSiStripClusterChargeCutNone'))
 #     return process
 
+def customiseFor21810(process):
+    from RecoLocalCalo.CaloTowersCreator.calotowermaker_cfi import calotowermaker
```
Comment (Martin-Grunewald) [inline HLTrigger/Configuration/python/customizeHLTforCMSSW.py]: Why is this import needed? Nothing from ```calotowermaker``` seems to be used here.

**Comment by abdoulline** [comment]
Ah, RelVals... OK. But I thought in RelVals - still HLT menue(s) are 
used, regardelss of 2018 era (with new thresholds for CaloTowers) which 
effectively acts only on Offline RECO.

**Comment by deguio** [comment]
superseded by the last commit in #21842
